tt_metal/hw/inc/blackhole/eth_interface.h

```diff
@@ -0,0 +1,7 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "tt_metal/third_party/umd/src/firmware/riscv/blackhole/eth_interface.h"
```

Comment:
does this file exist in hw?  I hate adding another external dependency in the firmware

Reply:
No...I'd like to go the other way: put the contents of umd's eth_interface.h into the hw/inc copy so that hw doesn't have external dependencies.  Ideally, umd would then include that.

Reply:
after speaking offline, it seems like the "proper" fix would be to restructure firmware headers and potentially pull it out into a separate repo. 

Decided to revisit this at time of restructuring